include/updatehub.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ *
+ *  @brief UpdateHub Firmware Over-the-Air for Zephyr Project.
+ */
+
+#ifndef _UPDATEHUB_H_
+#define _UPDATEHUB_H_
+
```

Comment:
> I am just pondering that should the include file be placed into `include/net`, what do you think?

I am not sure I buy the net change. Mainly because in future we could have other ways of putting a new firmware on the device, for example using a pendrive (for the devices which has the connector). 

So I'd prefer to not limit to the net namespace.

Reply:
Fair enough, I am fine if it is not part of network subsystem.